### DISCUSSION TITLE
🚀 Feature Request

### DISCUSSION THREAD

**Discussion by doc-jay** [louislam/dockge#224] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Ability to display the IP and port of a container on the left panel of the dashboard.  Also the ability to 'search' via the IP or port for a used container.  

### ✔️ Solution

Useful for adding new containers when specifying a static IP and port for a container to ensure it is not already in use.

### ❓ Alternatives

currently I need to use Portainer to display IP addresses and ports

**Comment by markhaines** [comment]
Came here to request this, would be super-useful when you are managing a few tens of containers trying to keep track of which ports are in use.

**Comment by Dvalin21** [comment]
For users of Synology NAS, its essential as we need to go into the firewall and allow specific docker ip address at times to get the dockers to work. Including the docker ip for dockge.
